### DISCUSSION TITLE
Ασκήσεις γραμμής εντολών

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1241] [Q&A] [answered]
Πόσες ακριβώς ασκήσεις πρέπει να κάνουμε;

**Comment by p20sami** [comment]
Θα ήθελα να συμπληρώσω την ερώτηση, εκτός από το 6ο παραδοτεο, οι υπόλοιπες ασκησεις γραμμής εντολών πρέπει να ειναι της λίστας sw η warm up?

**Reply by nikosbout** [reply]
Οι ασκήσεις γραμμών εντολών πρέπει να ανήκουν στην λίστα SW σύμφωνα με τις οδηγίες. Οι warm up είναι απλώς για εξικοίωση και δεν βαθμολογούνται (όπως και στο μάθημα HCI).

**Comment by axilleasmandravelis** [comment, marked as answer]
> Πόσες ακριβώς ασκήσεις πρέπει να κάνουμε;

Σύμφωνα με τις οδηγίες και τον πίνακα υπάρχουν  10 παραδοτέα ( 1 για κάθε εβδομάδα ).
Όσον αφορά τις ασκήσεις γραμμών εντολών είναι  συνολικά 3 στις εβδομάδες 2,3 και 6
